searx/engines/wikidata.py

```diff
@@ -641,7 +641,7 @@ def get_str(self, result, language):
             return get_external_url(url_id, value)
 
         if self.url_path_prefix:
-            [account, domain] = value.split('@', 1)
+            [account, domain] = [x.strip("@ ") for x in value.rsplit('@', 1)]
```

Comment:
```suggestion
            [account, domain] = [x.lstrip("@") for x in value.rsplit('@', 1)]
```
would be more applicable I think? We probably don't want to strip `@` from the end as we only add an `@` to the start.

Reply:
:thinking: sure? .. I think we don't want to have an `@` in `account` nor in `domain` .. this was at least my intention.

Reply:
We certainly don't want it in domain, but that's already assured by using `rsplit(...)`, since it splits at the last `@`.

`@` in `account` might be possible though if some service allow usernames like `foo@`. I don't know if there's any that does so though ...

Reply:
Though you're probably right that most services don't allow any `@` in the `account`, so it doesn't matter which solution we use in that case.